src/main/java/webApp/QuestionMessage.java

```diff
@@ -52,6 +57,12 @@ public String getQuestion() {
         return question;
     }
 
+    /*
+     * Retrieve the question options
+     * @returns {Collection<String>}
+     */
+    public Collection<String> getOptions() { return options; }
```

Comment:
This will also need to be tested in the test class I mentioned above.

Reply:
On it.